lib/cassia/device.rb

```diff
@@ -2,5 +2,12 @@ module Cassia
   class Device
     include Virtus.model
     attribute :mac, String
+    attribute :type, String
+    attribute :bdaddrs
+    attribute :chipId, Integer
+    attribute :handle, String
+    attribute :name, String
+    attribute :connectionState, String
+    attribute :id, String
```

Comment:
maybe we should translate `id` to `mac`. And then we won't need the `id` attribute